mqtt3/1.0.0/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,196 @@
+= MQTT3 Connector 1.0 - Mule 4
```

Comment:
MQTT Version 3.x Connector

Reply:
For this one I changed it to MQTT 3 Connector, because it is the short name for Anypoint Connector for MQTT 3 (MQTT 3 Connector) so we have to be consistent in the entire docs.